src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -19,6 +19,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Inherits AbstractSyntaxFactsService
         Implements ISyntaxFactsService
 
+        Public Shared ReadOnly Property [Default] As New VisualBasicSyntaxFactsService
```

Comment:
See examples at http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Workspace/Host/Mef/ExportLanguageServiceFactoryAttribute.cs,4bfcbce32d7c37d9,references for using `ExportLanguageServiceFactory`.

For the private constructor, just add 

```VB
Private Sub New()
End Sub
```

to this class.